GDJS/Runtime/runtimegame.ts

```diff
@@ -937,6 +942,17 @@ namespace gdjs {
       }
     }
 
+    dispose(): void {
+        if (this._isDisposed) {
+            return;
+        }
+
+        this._isDisposed = true;
```

Comment:
What is the aim of this check? Calling the dispose method once is the responsibility of whoever created the game.

Reply:
done